assets/scss/_base.scss

```diff
@@ -869,3 +869,14 @@ body.td-documentation {
       display: none;
     }
 }
+
+// nav-tabs and tab-content
+.nav-tabs {
+  border-bottom: none !important;
+}
+
+.tab-content {
+  padding: 0 16px 10px;
```

Comment:
See https://deploy-preview-26828--kubernetes-io-master-staging.netlify.app/docs/concepts/configuration/manage-resources-containers/#resource-emphemeralstorage-consumption

When I remove the padding-bottom, the bottom border will be too close to the content. Is it ok?
![屏幕快照 2021-03-10 上午11 03 14](https://user-images.githubusercontent.com/7345199/110570007-3e929680-8190-11eb-9970-5796c274bc05.png)

Reply:
> We may want to reset the padding for `highlight` inside `tb-content` to zero as well. See preview link for the effect.
> 
> ```
> .td-content .highlight {
>     margin: 1rem 0;
>     padding: 0;
> }
> ```

Do you mean to reset the padding or the margin.

Reply:
> When I remove the padding-bottom, the bottom border will be too close to the content. Is it ok?

I agree, this is very tight without the padding. What do other similar elements use?

Reply:
I was proposing to remove that margin because it is sometimes generating silly output like these.

<img width="915" alt="Screen Shot 2021-03-11 at 9 14 58 AM" src="https://user-images.githubusercontent.com/5094756/110720955-72ca8d80-824a-11eb-874c-96d5b289aabd.png">

<img width="766" alt="Screen Shot 2021-03-11 at 9 15 27 AM" src="https://user-images.githubusercontent.com/5094756/110721016-9261b600-824a-11eb-8afe-ca4ea9922d72.png">

Reply:
@tengqm @nate-double-u I push a new commit, which remain the tab-content padding-bottom, but remove the margin of the hightlight inside the tab-content. [Have a look](https://deploy-preview-26828--kubernetes-io-master-staging.netlify.app/docs/tasks/tools/install-kubectl/)

Reply:
> I was proposing to remove that margin because it is sometimes generating silly output like these.

I agree, that's too much padding. I am seeing similar padding on Notes, which may be a separate issue

<img width="866" alt="Screen Shot 2021-03-10 at 6 09 05 PM" src="https://user-images.githubusercontent.com/4453979/110725162-f09b8400-81cb-11eb-8664-c24c506f83b1.png">

Reply:
> @tengqm @nate-double-u I push a new commit, which remain the tab-content padding-bottom, but remove the margin of the hightlight inside the tab-content. [Have a look](https://deploy-preview-26828--kubernetes-io-master-staging.netlify.app/docs/tasks/tools/install-kubectl/)

That's better, but it still feels tight on the bottom -- could you make the bottom padding the same as the left, right, and top? (they all look the same, or very close to the same)

<img width="773" alt="Screen Shot 2021-03-10 at 6 12 23 PM" src="https://user-images.githubusercontent.com/4453979/110725297-39533d00-81cc-11eb-9a91-6765124a28a6.png">

Reply:
Could be the same. @nate-double-u The default padding for `highlight` is too much. Not sure if there is a separate knob for it.

Reply:
> > @tengqm @nate-double-u I push a new commit, which remain the tab-content padding-bottom, but remove the margin of the hightlight inside the tab-content. [Have a look](https://deploy-preview-26828--kubernetes-io-master-staging.netlify.app/docs/tasks/tools/install-kubectl/)
> 
> That's better, but it still feels tight on the bottom -- could you make the bottom padding the same as the left, right, and top? (they all look the same, or very close to the same)
> 
> <img alt="Screen Shot 2021-03-10 at 6 12 23 PM" width="773" src="https://user-images.githubusercontent.com/4453979/110725297-39533d00-81cc-11eb-9a91-6765124a28a6.png">

already make the bottom padding same with the left and right

Reply:
Thanks, I see it now

<img width="773" alt="Screen Shot 2021-03-10 at 6 12 23 PM" src="https://user-images.githubusercontent.com/4453979/110726793-e62eb980-81ce-11eb-8e04-e214ea6ec9a6.png">